wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -117,6 +118,7 @@ public WorkspaceService(
       WorkspaceLinksGenerator linksGenerator,
       @Named(CHE_WORKSPACE_PLUGIN_REGISTRY_URL_PROPERTY) @Nullable String pluginRegistryUrl,
       @Named(CHE_WORKSPACE_DEVFILE_REGISTRY_URL_PROPERTY) @Nullable String devfileRegistryUrl,
+      @Named("che.infra.kubernetes.pvc.enabled") boolean defaultPersistVolumes,
```

Comment:
My understanding is that this property is used to *disable* PVCs on a Che server deployment; in, e.g. OpenShiftEnvironmentProvisioner, we have the check:
```java
    if (pvcEnabled) {
      volumesStrategy.provision(osEnv, identity);
    }
```
so it seems as though setting this property to `false` would disable all PVC provisioning.

Reply:
Though that also raises the question of whether workspaces work *at all* without it set, since at bare minimum `/projects` needs to be shared between Theia and the main container.

Reply:
Thanks for catching it!
Now it seems to be that `che.infra.kubernetes.pvc.enabled` is not really usable property, maybe it makes sense to deprecate it in favor of new `che.workspace.persist_volumes.default`... Will take a look